packages/tools/webpack-fluid-loader/src/loader.ts

```diff
@@ -281,7 +282,8 @@ export async function start(
             );
             assert(prefetched, 0x1eb /* "Snapshot should be prefetched!" */);
         }
-        container1 = await loader1.resolve({ url: documentUrl });
+        container1 = await loader1.resolve({ url: documentUrl, headers: {[LoaderHeader.loadMode]: {deltaConnection: "none"}}});
```

Comment:
It might be not clear from this code why you are making this change, especially that it looks like general piece of code used outside of bundle size tests - someone can easily change / revert it. So I'd put some comments here

Reply:
It is just to use the dynamic load in other apps like todo, smde which uses webpack fluid loader.

Reply:
It is not necessary for the bundle functionality, It is just to see it our examples. I will add a comment though.